README.md

```diff
@@ -6,11 +6,11 @@
 
 [![npm version](https://badge.fury.io/js/dc-delivery-sdk-js.svg)](https://badge.fury.io/js/dc-delivery-sdk-js)
 
-This sdk is designed to help build client side and server side content managed applications.
+This SDK is designed to help build client side and server side content managed applications.
 
 ## Features
 
-- Fetch content and slots using the [Content Delivery 1](https://docs.amplience.net/integration/deliveryapi.html#the-content-delivery-api) and/or [Content Delivery 2](https://docs.amplience.net/development/contentdelivery/readme.html)
+- Fetch content and slots using the [Content Delivery 1](https://docs.amplience.net/integration/deliveryapi.html#the-content-delivery-api) and/or [Content Delivery 2](https://docs.amplience.net/development/contentdelivery/readme.html) or fetch fresh content and slots for use with SSG build tools using [Fresh API](https://amplience.com/docs/development/freshapi/fresh-api.html)
```

Comment:
Could be more consistent with the use of 'the' and 'API'. Might be better to make separate bullet points like so:

- Fetch content and slots using the [Content Delivery 1 API](https://docs.amplience.net/integration/deliveryapi.html#the-content-delivery-api) or the [Content Delivery 2 API](https://docs.amplience.net/development/contentdelivery/readme.html) 
- Fetch fresh content and slots for use with SSG build tools using the [Fresh API](https://amplience.com/docs/development/freshapi/fresh-api.html)